Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/RayTracing/RayTracingMaterialUtils.azsli

```diff
@@ -26,7 +26,7 @@ TextureData GetHitTextureData(RayTracingMaterialSrg::MaterialInfo materialInfo,
     if (materialInfo.m_textureFlags & TEXTURE_FLAG_BASECOLOR)
     {
         // array index of the baseColor texture for this material in the m_materialTextures unbounded array
-        uint baseColorTextureArrayIndex = RayTracingMaterialSrg::m_materialTextureIndices[materialInfo.m_textureStartIndex + MATERIAL_BASECOLOR_TEXTURE_OFFSET];
+        uint baseColorTextureArrayIndex = RayTracingMaterialSrg::m_materialTextureIndices[NonUniformResourceIndex(materialInfo.m_textureStartIndex + MATERIAL_BASECOLOR_TEXTURE_OFFSET)];
```

Comment:
Not an issue atm but NonUniformResourceIndex probably doesnt work on mobile hw. Something to keep in mind.